src/app/blogpost/post-list/post-list.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="post-list">
+  <div class="post-list--container">
+    <div class="post-list--heading">
+      <h1>Posts</h1>
+    </div>
+
+    <div class="post-list--list">
+      <div *ngIf="posts$ | async">
```

Comment:
it's possible to cache posts
`posts$ | async as posts`
and later re-use 
`*ngFor="let post of posts"`